geometry_msgs/Pose2D input
---
open_base/Velocity output
